mapie/classification.py

```diff
@@ -388,22 +411,19 @@ def _get_last_index_included(
 
         Returns
         -------
-        NDArray of shape (n_samples, n_classes)
+        NDArray of shape (n_samples, n_alpha)
             Index of the last included sorted probability.
         """
         if (
             (include_last_label is True) or
```

Comment:
(include_last_label is True) -> include_last_label
no need of "is True"

Reply:
done